build.gradle

```diff
@@ -1,39 +1,79 @@
+import java.nio.file.Files
+import cuchaz.enigma.command.ConvertMappingsCommand
+buildscript {
+    repositories{
+		maven {
+			name "Fabric Repository"
+			url 'https://maven.fabricmc.net'
+		}
+		mavenLocal()
+    }
+
+    dependencies {
+		classpath "cuchaz:enigma:0.14.1-local"
+		classpath "net.fabricmc:stitch:0.2.1-local"
```

Comment:
What is this used for? cannot see any inports for it?

Reply:
removed